Makefile

```diff
@@ -39,7 +39,7 @@ style_tests:
 system_tests:
 	pipenv run behave system_tests/features # This will only run the system tests
 
-acceptance_tests: system_tests
```

Comment:
I found running the system_tests before running the acceptance_tests very useful, any reason for getting rid of it?

Reply:
Not getting rid of it, just separating them from the acceptance tests as it surely makes more sense to run them before setup and after the health check has been performed.